include/seqan/platform.h

```diff
@@ -401,7 +401,11 @@ typedef int8_t __int8;     // nolint
 
 // A macro to eliminate warnings on GCC and Clang
 #if defined(COMPILER_GCC) || defined(COMPILER_CLANG) || defined(COMPILER_LINTEL)
-#define SEQAN_UNUSED __attribute__((unused))
```

Comment:
This might be a problem, because you can place `__attribute__` at different places/positions than `[[maybe_unused]]`

Reply:
Just mentioning:

```c++
using type = type __attribute__(); // valid, [] invalid
using type [maybe_unused] = type; // valid

Reply:
I know, but the only place that could cause this trouble was in the declaration of the exception handler, which should be fixed by this PR as well. But we have to wait for the builds to be sure.